.scalafmt.conf

```diff
@@ -0,0 +1,16 @@
+maxColumn = 80
+
+style = IntelliJ
```

Comment:
I don't feel overly biased for either `Intelij` format or `default` but i would like which ever one you choose to be applied to the template.